docs/contributing-to-analyses/pr-review-and-merge/respond-to-review.md

```diff
@@ -0,0 +1,167 @@
+<!-- comment to force two grid columns -->
+
+- **`Checks`**
+    - This tab shows results from certain types of code checks that have been automatically set up by the Data Lab; you can ignore this tab.
+- **`Files changed`**
+    - This tab shows a line-by-line overview of content you have added and removed in your branch.
+    Changes shown here should reflect the difference between files in your branch and the base branch your PR is targeting.
+This should be the `main` branch of the `AlexsLemonade/OpenScPCA-analysis` repository.
+    - Referring to this tab is a great way to quickly see all your changes, as well as respond to [reviewer suggestions or comments](#inline-comments).
+
+</div>
+
+
+## Responding to review comments
+
+When making code changes in response to review, you need to make sure your local repository is staying in-sync with any changes made directly on GitHub.
+Follow these steps when responding to review to avoid [merge conflicts](../creating-pull-requests/resolve-merge-conflicts.md):
+
+1. Before making any code changes, _pull_ your branch from GitHub.
+This will ensure you are up-to-date with any code changes made on GitHub, including committed review suggestions or updates from merging the upstream `main` branch into your pull request branch.
+    <figure markdown="span">
+        ![Pull in GitKraken.](../../img/respond-to-review-pull.png){width="600"}
+    </figure>
+
+1. Make changes, commit, and push as you normally would.
+Any changes you push to the branch with an open PR will automatically be reflected in the PR.
+    - When committing changes in response to review, the smaller the better!
+    Review will go more smoothly if each commit addresses only one review comment.
+
+
+## Re-requesting review
+
+Once you feel that you have addressed your reviewer's comments, you can [re-request review by clicking the cycle icon next to their handle on the right side of the PR page](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#re-requesting-a-review).
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-x.png){width="400"}
+</figure>
+
+You can also post a comment in your PR summarizing your response to review, which will help your reviewer navigate your code changes:
+
+- Which review comments did you address?
+    - Is there more information you can tell your reviewer that will help them understand your code changes?
+- Which review comments, if any, did you not address?
+    - Was it because the review comment itself was confusing, or because you weren't sure how to address it?
+    Communicating this with your reviewer will help you both out!
+
+## Types of reviewer comments
+
+When reviewing your pull request, there are several types of comments your reviewer may leave for you to respond to.
+<!-- Please refer to [our documentation on example reviews](STUB_LINK example reviews). -->
```

Comment:
I think we can remove this and just have examples here. I would move this whole section to be before `Responding to review comments`.